gymnasium/envs/mujoco/humanoid_v5.py

```diff
@@ -0,0 +1,516 @@
+
+    Additionally, after all the positional and velocity based values in the table,
+    the observation contains (in order):
+    - *cinert:* Mass and inertia of a single rigid body relative to the center of mass
+    (this is an intermediate result of transition). It has shape 13*10 (*nbody * 10*)
+    and hence adds to another 130 elements in the state space.
+    - *cvel:* Center of mass based velocity. It has shape 13 * 6 (*nbody * 6*) and hence
+    adds another 78 elements in the state space
+    - *qfrc_actuator:* Constraint force generated as the actuator force. This has shape
+    `(17,)`  *(nv * 1)* and hence adds another 17 elements to the state space.
+    - *cfrc_ext:* This is the center of mass based external force on the body.  It has shape
+    13 * 6 (*nbody * 6*) and hence adds to another 78 elements in the state space.
+    where *nbody* stands for the number of bodies in the robot and *nv* stands for the
+    number of degrees of freedom (*= dim(qvel)*)
+
+    The body parts are:
+
+    | id (for `v2`, `v3`, `v4)` | id (for `v5`) | body part |
+    | ---|  ---   |  ------------  |
+    | 0  |excluded| worldbody (note: all values are constant 0) |
+    | 1  | 0      | torso |
+    | 2  | 1      | lwaist |
+    | 3  | 2      | pelvis |
+    | 4  | 3      | right_thigh |
+    | 5  | 4      | right_sin |
+    | 6  | 5      | right_foot |
+    | 7  | 6      | left_thigh |
+    | 8  | 7      | left_sin |
+    | 9  | 8      | left_foot |
+    | 10 | 9      | right_upper_arm |
+    | 11 | 10     | right_lower_arm |
+    | 12 | 11     | left_upper_arm |
+    | 13 | 12     | left_lower_arm |
+
+    The joints are:
+
+    | id (for `v2`, `v3`, `v4)` | id (for `v5`) | joint |
+    | ---|  ---   |  ------------  |
+    | 0  |excluded| root (note: all values are constant 0) |
+    | 1  |excluded| root (note: all values are constant 0) |
+    | 2  |excluded| root (note: all values are constant 0) |
+    | 3  |excluded| root (note: all values are constant 0) |
+    | 4  |excluded| root (note: all values are constant 0) |
+    | 5  |excluded| root (note: all values are constant 0) |
+    | 6  | 0      | abdomen_z |
+    | 7  | 1      | abdomen_y |
+    | 8  | 2      | abdomen_x |
+    | 9  | 3      | right_hip_x |
+    | 10 | 4      | right_hip_z |
+    | 11 | 5      | right_hip_y |
+    | 12 | 6      | right_knee |
+    | 13 | 7      | left_hip_x |
+    | 14 | 8      | left_hiz_z |
+    | 15 | 9      | left_hip_y |
+    | 16 | 10     | left_knee |
+    | 17 | 11     | right_shoulder1 |
+    | 18 | 12     | right_shoulder2 |
+    | 19 | 13     | right_elbow|
+    | 20 | 14     | left_shoulder1 |
+    | 21 | 15     | left_shoulder2 |
+    | 22 | 16     | left_elfbow |
+
+    The (x,y,z) coordinates are translational DOFs while the orientations are rotational
```

Comment:
DOFs?

Reply:
"Degrees of Freedom" a very common term in robotics